__tests__/accountAbstraction.test.ts

```diff
@@ -0,0 +1,461 @@
+// tests of handling of accounts including account abstraction about hashes and signatures
+import {
+  AbstractionFunctions,
+  Account,
+  ArraySignatureType,
+  BigNumberish,
+  Call,
+  CallData,
+  Calldata,
+  Contract,
+  DeclareSignerDetails,
+  DeployAccountSignerDetails,
+  Provider,
+  RawCalldata,
+  Signature,
+  Signer,
+  TypedData,
+  cairo,
+  constants,
+  ec,
+  encode,
+  hash,
+  typedData,
+} from '../src';
+import {
+  compiledAccountAbstractionCasm,
+  compiledAccountAbstractionSierra,
+  compiledDummy1Casm,
```

Comment:
what do `Dummy1` and `Dummy2` mean?

Reply:
2 tests are made to test __declare__ signature. They are used just to test if a dummy contract (whatever it contains) can be declared with the right abstracted signature. So I took the 2 smallest contracts in my contract bank and I use them for this test.